src/plugins/intel_cpu/src/emitters/snippets/brgemm_base.cpp

```diff
@@ -248,6 +248,11 @@ void BrgemmBaseKernelExecutor::update_config(const ov::snippets::lowered::Expres
             beta = get_beta(loop_manager, static_cast<int>(loop_ids.back()), current_expanded_loop_info);
     }
 
+#ifndef OPENVINO_ARCH_X86_64
+    config.update(DIM_CAST(M), DIM_CAST(N), DIM_CAST(K), 0, 0, 0, beta);
+    return;
+#endif
+
```

Comment:
I'd say that there should be common cross-arch base class with method `init_runtime_params(M,N,K,LDA,LDB,LDC)`.
Then x64 dnnl executors update `LDB` and `LDA` if needed. aarch64 (tpp) should call `update_config(...)` with these parameters.

If we use `ifdef` in common code, I think this is a sign of problematic code and we should resolve it.

Reply:
The base class is created in `src/plugins/intel_cpu/src/emitters/snippets/brgemm_base.hpp` which has M, N, K, beta, LDA, LDB and LDC that are common for TPP and onednn. TPP and onednn have their own static params and hash as different type and params needed.  Thanks Alexandra!